tests/test_item_search.py

```diff
@@ -653,8 +653,24 @@ def test_result_paging(self) -> None:
         # is requested
         pages = list(search.pages())
 
-        assert pages[1] != pages[2]
-        assert pages[1].items != pages[2].items
+        assert pages[0] != pages[1]
+        assert pages[0].items != pages[1].items
```

Comment:
This was a invalid test in The Old Way, that we didn't catch because we didn't update our cassettes.